src/UndockedRegFreeWinRT/Nuget/Microsoft.Windows.UndockedRegFreeWinrt.nuspec

```diff
@@ -24,6 +24,7 @@
     <file src="$winrtactx64debug$" target="runtimes\x64\debug\"/>
     <file src="$winrtactx86release$" target="runtimes\x86\release\"/>
     <file src="$winrtactx86debug$" target="runtimes\x86\debug\"/>
+    <file src="$mwinrtactx64release$" target="lib\netstandard2.0"/>
```

Comment:
is there a reason for not including an x86 version as well? Or is this really "any cpu"?

Reply:
I don't really understand how it's working but when I tested it, it seems to work on any CPU. I don't know if I am doing this nuspec part properly by targeting "lib\netstandard2.0". If you know someone who is expert in this, let me know.

Reply:
I believe this is what you want. Targeting .Net standard 2.0 will provide compat with a wide range of .NET targets, if I understand correctly. You can find more about it here: https://docs.microsoft.com/en-us/nuget/reference/target-frameworks.